Sources/HipMobileUI/Pages/LoadingPage.xaml

```diff
@@ -33,32 +33,33 @@
+                    
+                    <StackLayout Grid.Row="0" Orientation="Vertical" Spacing="0" HorizontalOptions="CenterAndExpand" Padding="20" Margin="15"
                                  VerticalOptions="CenterAndExpand">
-                        <Label Text="{Binding Text}" HorizontalTextAlignment="Center" FontSize="20"
-                               FontAttributes="Bold" Style="{DynamicResource Ssp-Bold}" />
-                        <Label Text="{Binding Subtext}" HorizontalTextAlignment="Center" FontSize="16" Style="{DynamicResource Ssp-Regular}"/>
-                        <ProgressBar Progress="{Binding LoadingProgress}" Margin="20,0">
+                        <Label Text="{Binding Text}" HorizontalTextAlignment="Center" FontSize ="20"
+                               FontAttributes="Bold" Style="{DynamicResource Ssp-Bold}"/>
+                        <Label Text="{Binding Subtext}" HorizontalTextAlignment="Center" FontSize="16" Style="{DynamicResource Ssp-Regular}" Margin="10"/>
+                        <ProgressBar Progress="{Binding LoadingProgress}">
                             <ProgressBar.Effects>
                                 <effects:ProgressBarColorEffect Color="{StaticResource PrimaryDarkColor}" />
                             </ProgressBar.Effects>
                         </ProgressBar>
-                        <Label Text="{Binding LoadingProgress, StringFormat='{0:p0}'}" HorizontalTextAlignment="Center"
-                               FontSize="16" Style="{DynamicResource Ssp-Regular}" />
+                        <Label Text="{Binding LoadingProgress, StringFormat='{0:p0}'}" HorizontalTextAlignment="Center" Margin="5"
+                               FontSize="Default" Style="{DynamicResource Ssp-Regular}" />
                     </StackLayout>
-                    <Image Grid.Row="1" Source="hiphop_transparent">
+                    <Image Grid.Row="1" Source="hiphop_transparent" VerticalOptions="Center" HorizontalOptions="CenterAndExpand" >
                     </Image>
-                    <StackLayout Grid.Row="2" Orientation="Vertical" HorizontalOptions="FillAndExpand">
-                        <BoxView HeightRequest="1" Color="{StaticResource DarkGrayColor}" />
-                        <Label Text="{helpers:Translate Cancel}" FontSize="Large" TextColor="{StaticResource PrimaryDarkColor}"
-                               HorizontalOptions="CenterAndExpand" VerticalOptions="CenterAndExpand" />
-                        <StackLayout.GestureRecognizers>
-                            <TapGestureRecognizer Command="{Binding CancelCommand}" />
-                        </StackLayout.GestureRecognizers>
+                    
+                    <StackLayout Grid.Row="2" Orientation="Vertical" HorizontalOptions="CenterAndExpand" VerticalOptions="CenterAndExpand">
+                        <Button x:Name="CancelButton"
```

Comment:
Since the `Button` is the only child of its container (`StackLayout`), it may be possible to remove the `StackLayout` and only leave the `Button`. You could try removing the enclosing `StackLayout` and giving the `Button` the following attributes:

- `Grid.Row="2"`
- `HorizontalOptions="Center"`
- `VerticalOptions="End"` (not entirely sure about this one, maybe also `CenterAndExpand`)

Reply:
Some attributes of button is now changed and button is not in "StackLayout" anymore. Also If VerticalOption="End", the button is touching the margin of the boundary box